src/main/java/ch/njol/skript/conditions/CondIsOnline.java

```diff
@@ -33,32 +33,41 @@
  * @author Peter Güttinger
  */
 @Name("Is Online")
-@Description("Checks whether a player is online.")
+@Description("Checks whether a player is online. The 'connected' version will return false once this player leaves the server, even if they rejoin.")
 @Examples({"player is online",
-		"player-argument is offline"})
+	"player-argument is offline",
```

Comment:
Hate that github won't let me make a suggestion but the `@Examples({""`, needs to move `player is online` down a line

Reply:
Can you explain that again I'm not sure what you mean

Reply:
```
@Examples({
		"",
		"",
		"",
})

```
Unless github's diff is messed up it looks like you still have it
```
@Examples({"player is online"
})
```

Reply:
<img width="412" alt="image" src="https://github.com/SkriptLang/Skript/assets/14147477/9e83b982-efb9-43ad-98cb-9f05100f1781">